doc/how_to_guides/how_to_setup_docker_containers_in_ubuntu.rst

```diff
@@ -0,0 +1,84 @@
+How to Set Up MoveIt 2 Docker Containers in Ubuntu
+===================================================
+This guide will provide a walkthrough on how to get a Docker container with MoveIt 2 dependencies set up quickly.
+It includes a script that will get you up and running in MoveIt quickly!
+This guide is intended for people who would like to have a separate environment for working with MoveIt up and running quickly \
+without having to do much configuring. In this guide, we will be setting up a ROS2 Galactic environment.
+
+Learning Objectives
+-------------------
+
+- How to setup a Docker environment using the provided script
+
+Requirements
+------------
+
+- Ubuntu 20.04
+- `Docker Installation for Ubuntu <https://docs.docker.com/engine/install/ubuntu/>`_
+- `Nvidia drivers for Docker <https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#setting-up-nvidia-container-toolkit>`_
+
+Steps
+-----
+1. Install Docker (a link is available in the Requirements section) and be sure to follow the `Linux Post Install <https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user>`_ instructions. If you do not complete these additional steps you will need to preface all ``docker`` commands with ``sudo``.
+
+2. Open a terminal session and create an empty text file.
+
+  .. code-block:: bash
+
+    mkdir -p ~/Docker/scripts
```

Comment:
Would it be much more standard to just put this in `.local/bin` or some similar directory? Then you don't need to specify the full file path later... that or maybe you could create a repo with this file if you expect to update it in the future and people could put it in `~/opt` or something similar and then they could update it when you push updates.